.circleci/config.yml

```diff
@@ -28,10 +28,13 @@ jobs:
       - checkout
       - run:
           name: Configure hokusai
-          command: hokusai configure --kubectl-version 1.6.3 --s3-bucket artsy-citadel --s3-key k8s/config --platform linux
+          command:
+            hokusai configure --kubectl-version 1.10.7 --s3-bucket artsy-citadel
+            --s3-key k8s/config --platform linux
```

Comment:
Is this line-break intentional and valid, or auto-formatting gone wrong? There's a similar one below.

Reply:
Checked this out and loaded it successfully so it seems valid.